static/app/components/demo/demoHeader.tsx

```diff
@@ -61,7 +63,7 @@ const Wrapper = styled('div')`
   }
 `;
 
-const StyledLogoSentry = styled(LogoSentry)`
```

Comment:
Can we change `LogoSentry` to pass `className` prop to the svg element instead?

Reply:
@billyvg but then wouldn't I have to pass `className` as a prop? Not sure how this works as a styled component.

Reply:
How this works (at a high level) is that during the build process...

```
styled(Component)`
...[css]...
`
```

becomes a class name string (and has the CSS injected to dom) and then is passed to `Component`. Which means that `Component` must pass `className` prop down to the appropriate element.